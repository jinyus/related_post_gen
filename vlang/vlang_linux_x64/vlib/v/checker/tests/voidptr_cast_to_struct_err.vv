struct Foo {}

fn main() {
	a := Foo(unsafe { nil })
	println(a)
}
