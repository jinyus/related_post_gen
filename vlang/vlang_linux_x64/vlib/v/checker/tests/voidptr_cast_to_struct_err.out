vlib/v/checker/tests/voidptr_cast_to_struct_err.vv:4:7: error: cannot cast `voidptr` to struct
    2 | 
    3 | fn main() {
    4 |     a := Foo(unsafe { nil })
      |          ~~~~~~~~~~~~~~~~~~~
    5 |     println(a)
    6 | }
